model/task.graphql

```diff
@@ -1,15 +1,50 @@
-""" 
+"""
 @model
 """
 type Task {
-  id: ID!
+  """
+  @db.primary
+  """
+  _id: ObjectID!
   title: String!
   description: String!
   status: TaskStatus
+  type: String
+  priority: Int
+  public: Boolean
+  status: TaskStatus
+  startDate: Date
+  payload: JSON
+
+  """
+  @oneToOne
+  """
+  location: [Address]
+}
+
+"""
+@model
+@crud.delete: false
```

Comment:
Unrelated, but idea. Crud config could be merged into the model annotation

Reply:
@craicoverflow you telling me that we have annotation support already?? 🔥 🔥 🔥

Reply:
LEt's create issue for that! good idea